src/sample/stuDashController.java

```diff
@@ -454,10 +446,10 @@ public void validateCGPA(KeyEvent e) {
         }
       }
     } else {
-      Messages.setText("CGPA cannot be empty!");
-      saveButton.setDisable(true);
-      name.setDisable(true);
-      nric.setDisable(true);
+      //Messages.setText("CGPA cannot be empty!");
+      saveButton.setDisable(false);
+      name.setDisable(false);
+      nric.setDisable(false);
```

Comment:
why false?

Reply:
because the cgpa format is correct le, so they can save their info or edit their name and nric